app/templates/project-name.Gulp/tasks/styles.js

```diff
@@ -1,15 +1,16 @@
 import gulp from 'gulp';
 import sourcemaps from 'gulp-sourcemaps';
-import rename from 'gulp-rename';
 import plumber from 'gulp-plumber';
 import colors from 'ansi-colors';
 import log from 'fancy-log';
 import sass from 'gulp-sass';
 import postcss from 'gulp-postcss';
 import config from '../config';
 import autoprefixer from 'autoprefixer';
-import postcsstype from 'postcss-responsive-type';
-import postcssclear from 'postcss-clearfix';
+import postcssClear from 'postcss-clearfix';
```

Comment:
Is anyone actually using this? 😄 I believe it's much easier to just create a class which you can reuse everywhere instead of duplicating the code. What do you think?

Reply:
💇‍♂️ I think it's just legacy stuff.. for now I can only find it in _cookieconsent.scss for code which weren't being used anyway. Furthermore, I agree it's a lot clear to use a class and sice we have bootstrap included we get it for [clearfix](https://getbootstrap.com/docs/4.1/utilities/clearfix/) class and mixin for free thus I've commited code removing this dependency 👍